src/lib/customers/comet-group-de.json

```diff
@@ -0,0 +1,44 @@
+{
+	"name": "Comet",
+	"title": "bespinian und Comet entwickeln den IoT-Healthmonitor BLOX",
+	"customer": "Die <a href=\"https://www.citechsensors.com\" target=\"_blank\">Comet Group</a> ist ein Schweizer Unternehmen mit Sitz in Flamatt, das weltweit mehr als 1700 Mitarbeitende beschäftigt. Sie ist ein führendes Technologieunternehmen, das Innovationen in der Röntgen- und Hochfrequenzbranche vorantreibt. Der Geschäftsbereich Comet x-ray entwickelt und produziert Röntgenmodule, die eine zerstörungsfreie Prüfung von Materialien und Industriekomponenten ermöglichen. Die geprüften Teile stammen aus der Luft- und Raumfahrt, der Automobilindustrie, der Batterieindustrie, der Halbleiterindustrie und anderen Märkten.",
```

Comment:
Link redirects to `www.citechsensors.com`.

Reply:
Many thanks, @zhews!